pages/test/react-query-recoil/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import type { NextPage } from "next";
+import { useRecoilState } from "recoil";
+import { useMemberQuery, usePutMemberMutation } from "~/hooks/api";
+import { videoIdAtomState } from "~/store";
+import type { PutMemberReq } from "~/types/members";
+
+const TestReactQueryRecoilPage: NextPage = () => {
+  const [value, setValue] = useState<PutMemberReq>({
+    nickname: "",
+    profileUrl: "",
+  });
```

Comment:
요청에 필요한 타입은 언제든 변경될 수 있기 때문에 이렇게 하면 더 좋을 거 같아

이렇게 되면 PutMemeberReq은 해당 api에 한 곳에서만 필요한 타입이 되니 삭제해주는 게 좋은 interface가 될 거 같아

```suggestion
  const [value, setValue] = useState<Pick<Memeber, 'nickname' | 'profileUrl'>>({
    nickname: "",
    profileUrl: "",
  });
```

Reply:
아래 코멘트 작업하면서 같이 업데이트했어!